lib/embroider/update-manifest-from-html.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs');
+const findScriptSrcs = require('find-scripts-srcs-in-document');
+
+function getFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
+  return Array.isArray(filePath.match(scriptFileNameRegEx)) ? filePath.match(scriptFileNameRegEx)[0] : '';
+}
+
+module.exports = function updateManifestFromHtml(root) {
+  // package.json is not present in root location skip
+  // below flow as there is no package.json to update.
+  if (!fs.existsSync(root + '/package.json')) {
+    return;
+  }
+
+  const pkg = JSON.parse(fs.readFileSync(root + '/package.json', 'UTF8'));
+  const appFile = pkg.fastboot.manifest.appFiles[0];
+
+  // if app.js file exists skip reading from the index.html as
+  // it is classic build flow
```

Comment:
I thought the plan was to not detect classic/embroider, but to create a feature that classic and embroider could both use, put it behind a feature specific flag, to allow classic folks time to upgrade.

Reply:
Ah, I misunderstood ask at that time. Let me update to include a feature flag to read from index.html that supports both build flows.

Reply:
I Fixed this. Added a feature (non-embroider) specific to make it work for any build flow.

Thank you.